app/views/partials/theme/simple/items/show.liquid

```diff
@@ -35,7 +35,7 @@
         <picture class="object-cover w-full">
           <source srcset="{{ item.photos[0].photo.versions.uncropped_webp }}" type="image/webp" alt="{{ item.name }}">
           <source srcset="{{ item.photos[0].photo.versions.uncropped }}" type="image/jpeg" alt="{{ item.name }}">
-          <img src="{{ item.photos[0].photo.versions.uncropped }}" alt="{{ item.name }}">
+          <img class="break-words" src="{{ item.photos[0].photo.versions.uncropped }}" alt="{{ item.name }}">
```

Comment:
I dont think images have words ;)

Reply:
like so, but if the image doesn't display, we will see item.name, which is under the alt attribute and then the text will break